config.w32

```diff
@@ -19,6 +19,7 @@ if (PHP_MONGO != "no") {
   ADD_SOURCES(configure_module_dirname + "/batch", "delete.c insert.c update.c write.c", "mongo");
 
   AC_DEFINE('HAVE_MONGO', 1);
+  AC_DEFINE('HAVE_MONGO_OPENSSL', 1);

```

Comment:
33      ADD_FLAG("CFLAGS_MONGO", " /D HAVE_MONGO_SSLLIB=1");

I'm curious, what is this?

Reply:
I traced it back to https://github.com/mongodb/mongo-php-driver-legacy/commit/ec3b84c78f4dcb09d5b20b6d23736357daf5d305. Best I can tell, the constant is never used in the extension source.

Reply:
Note: PHPC actually does defer its `AC_DEFINE()` for SSL to library detection (irrespective of depending on the `openssl` extension): https://github.com/mongodb/mongo-php-driver/blob/c3777b5a9d7286480739f8fe6680383c7c4abfff/config.w32#L156-L166